openml-generic-r/README.md

```diff
@@ -0,0 +1,24 @@
+OpenML Provider for generic R models
+------------------------------------------------------------
+
+This module contains an OpenML provider for loading custom code that implements
+Feedzai's R API.
+
+The implemented "Classifier" should define the following methods: 
+
+```r
+# loads a model. This can be a no-op if there's no need to save state  
+loadModel <- function() {
+    stop("This must be implemented by a concrete provider")
+}
+
+# score the instance stored in the 'instance' global var and returns an array with the probability for each of the classes
+getClassDistribution <- function() {
+    stop("This must be implemented by a concrete provider")
+}
+
+# returns the predicted class of the instance stored in the 'instance' global var
+classify <- function() {
```

Comment:
Shouldn't we address https://github.com/feedzai/feedzai-openml-r/issues/17 first and make these functions take parameters before we go updating the documentation just to have to rewrite it all over again later?

Reply:
I'm not sure if the solution is to make this function to receive parameters or to add a `synchronized` keyword where necessary. It's not a big effort, so I'm fine with updating this in a few days. If #17 gets addressed before merging this, we'll have that into account